README.md

```diff
@@ -65,29 +65,73 @@ Implementation of the [fusion identity spec]
 
 # Replication
 
-Current replication is taking care of by [ssb-friends] based on follow
-or block relations as described in that repo.
+Current replication is taken care of by [ssb-friends] and 
+[ssb-replication-scheduler] based on follow or block relations as 
+described in the latter repo.
 
 With meta feeds, contact messages on the main feed still form the
 basis for feed replication. From this basis the parts of the meta feed
 needed for the application should also be replicated. 
 
 Assuming one wants to do partial replication of a subset of a feed,
-first one tries to find the meta feed using the rpc getSubset from
-[subset replication] on the main feed author and type
-'metafeed/announce'. Then downloading the meta feed to try and find
-index feeds.
-
+the following steps should apply.
+
+**1. Discover and replicate the meta feed.** Given the main feed ID 
+(but not the main feed messages themselves) of a friend, we first need 
+to the root meta feed ID of that friend. To achieve that, we use the 
+RPC `getSubset` from [subset replication] on any connected peer, 
```

Comment:
Problem I just discovered: what if you already have gotten the meta feed ID of your friend in the past, but then restarted sbot, are you going to use `getSubset` *again* to re-discover it? It would be smarter that your sbot remembers the association between friend's main feed IDs and their root meta feed IDs.

Further, it doesn't seem sustainable to rely on `getSubset` for this, because then it means that a peer that wants to do partial replication *must* be connected to another peer that is doing full replication, because they will possess the `metafeed/announce` messages.

Thinking out loud: I know! We just need to `addOOO` the `metafeed/announce` message locally. That's perhaps important to amend in this PR.